ajax/libs/gibberish-aes/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gibberish-aes",
+  "filename": "gibberish-aes.min.js",
+  "version": "1.0.0",
+  "author": "Mark Percival <mark@mpercival.com>",
+  "description": "A lightweight Javascript Libray for OpenSSL compatible AES CBC encryption.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mdp/gibberish-aes.git"
+  },
+  "license": "MIT",
+  "npmName": "gibberish-aes",
```

Comment:
@iwdmb https://www.npmjs.com/package/gibberish-aes seems like not officially maintained. Could you please confirm this? If it's not officially maintained, we should use git auto-update config.